dev-data/about-course.data.tsx

```
@@ -225,8 +226,7 @@
       ...item,
       info: (
         <p>
-          Throughout the course, we mostly use
-          {' '}
+          Throughout the course, we mostly use{' '}
```

ESLint

[error] `{' '}` must be placed on a new line

(@stylistic/jsx-one-expression-per-line)